CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # CHANGELOG
 
+## Version 5.0.1, 2024-09-29
+
+### New Features (ENH)
+- Add `full` parameter to PdfWriter constructor (#2865)
+
+### Bug Fixes (BUG)
+- Update pyproject.toml with minimum Python version of 3.8 (#2859)
+- Cope with unbalanced delimiters in dictionary object (#2878)
+- Cope with encoding with too many differences (#2873)
+- Missing spaces in extract_text() method (#1328) (#2868)
+- Tolerate truncated files and no warning when jumping startxref (#2855)
+
+### Robustness (ROB)
+- Repair PDF with invalid Root object (#2880)
+- Continue parsing dictionary object when error is detected (#2872)
+- Merge documents with invalid pages in named destinations (#2857)
+- Tolerate comments(%) in arrays (#2856)
+
+### Documentation (DOC), Testing (TST), Code Style (STY), Developer Experience (DEV), Maintenance (MAINT)
+
+- (#2844), (#2862), (#2863), (#2847), (#2860), (#2867), (#2874), (#2879)
```

Comment:
Please do not do something like this - the grouping and listing PR numbers does not help anyone when viewing the changelog.

I personally would only include the MAINT and DEV cases here, but in their usual format.